contracts/clients/src/signer/index.ts

```diff
@@ -30,13 +30,16 @@ export async function initBlsWalletSigner({
   // properly initialized for all use cases, not just signing.
   const signerFactory = await signer.BlsSignerFactory.new();
 
+  const bundleDomain = getDomain(chainId, verificationGatewayAddress, "Bundle");
+  const walletDomain = getDomain(chainId, verificationGatewayAddress, "Wallet");
+
   return {
     aggregate,
-    getPublicKey: getPublicKey(signerFactory, domain),
-    getPublicKeyHash: getPublicKeyHash(signerFactory, domain),
-    getPublicKeyStr: getPublicKeyStr(signerFactory, domain),
-    sign: sign(signerFactory, domain, chainId),
-    signMessage: signMessage(signerFactory, domain),
-    verify: verify(domain, chainId),
+    getPublicKey: getPublicKey(signerFactory, bundleDomain),
+    getPublicKeyHash: getPublicKeyHash(signerFactory, bundleDomain),
+    getPublicKeyStr: getPublicKeyStr(signerFactory, bundleDomain),
+    sign: sign(signerFactory, bundleDomain),
+    signMessage: signMessage(signerFactory, walletDomain),
```

Comment:
I'm not 100% sure I got the 2 domains right here for all the functions.  All the tests pass with this set up though.